infrastructure/helm-chart/templates/configmap.yaml

```diff
@@ -4,7 +4,15 @@ metadata:
   name: hostnames
   namespace: {{ .Values.global.kubernetes.namespace }}
 data:
-  HOST: {{ .Values.global.kubernetes.host }}
+  {{ if .Values.global.ingress }}
+   {{ if .Values.global.ingress.https }}
+  HOST: https://{{ .Values.global.kubernetes.host }}
+   {{ else }}
+  HOST: http://{{ .Values.global.kubernetes.host }}
+   {{ end }}
+  {{ else }}
+  HOST: http://{{ .Values.global.kubernetes.host }}
```

Comment:
why is this line here twice?

Reply:
The logic is:
```
if the ingress section exists
  if ingress.https == true
    host: https...
  else (ingress.https != true)
    host: http...
else (ingress section doesn't exist)
  host: http...
```

Reply:
ah because the evaluation of `if .Values.global.ingress.https` would fail if `if .Values.global.ingress` isn't there